test/ducks/apps/__snapshots__/index.spec.js.snap

```diff
@@ -17,6 +17,54 @@ Array [
   },
   Object {
     "apps": Array [
+      Object {
```

Comment:
nit: Perhaps we can remove the test that generates this case because we needed to check that collect wasn't displayed